sdk/src/loom/System.ls

```diff
@@ -0,0 +1,131 @@
+package loom
+{
+    import system.platform.Platform;
+    import system.platform.PlatformType;
+    
+    public class System
```

Comment:
The issue I ran into is with the inclusion of the delegates. It seems that because the class has native delegates, the class itself must be native, and native classes can't have any extra goodies inside of them.